lib/create.js

```diff
@@ -4,7 +4,12 @@ const path = require('path');
 const { validateName } = require('./nameUtil');
 const YMLUtil = require('./binarisYML');
 
-const templateDir = './functionTemplates/nodejs/';
+const templateDir = 'functionTemplates';
+const runtimeToTemplateDir = {
```

Comment:
Since we keep `binaris.yml` in the template dir, it might be simpler to just use the runtime name as the directory name and not alter it in run time. It creates a minor duplication of the `python2` and `pypy2` function code, but is simpler and more readable.

Reply:
FWIU, this is exactly the opposite of what @rylandg says here: https://github.com/binaris/binaris/pull/263/files#diff-da3e06b4a4c89048b62dc5efae8ba65fR34

Reply:
It's definitely not the opposite. I explicitly said,

> If each runtime had it's own template dir I would suggest hardcoding, but in lieu of that I would say have them all share a root binaris.yml template and update accordingly.

After reading @michaeladda opinion above I believe he put my position into more eloquent words. Each runtime should probably get it's own directory.

Reply:
Oh sorry @rylandg I misread your comment..